INSTALL

```diff
@@ -109,7 +109,7 @@ Prepare your system
  2. Install dtrace (to generate provider.h)
     sudo apt-get install systemtap-sdt-dev
  3. Install libdispatch pre-reqs
-    sudo apt-get install libblocksruntime-dev libkqueue-dev libbsd-dev
+    sudo apt-get install libblocksruntime-dev libbsd-dev
```

Comment:
IOW I think it's completely fine to be *able* to use the submodule, but that should be for development purposes only, unlike libpwq